src/build-scripts/circleci-installdeps.bash

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+#
+
+set -ex
+
+#dpkg --list
+
+time apt-get update
+
+time apt-get install -y git
+time apt-get install -y cmake
+time apt-get install -y g++
+# time apt-get install -y g++-4.8
+#time apt-get install -y g++-7
+#time apt-get install -y g++-8
+time apt-get install -y ccache
+# time apt-get install -y clang
+# time apt-get install -y llvm
+time apt-get install -y libboost-dev
+time apt-get install -y libboost-thread-dev
+time apt-get install -y libboost-filesystem-dev
+time apt-get install -y libboost-regex-dev
+time apt-get install -y libtiff-dev
+time apt-get install -y libilmbase-dev
+time apt-get install -y libopenexr-dev
+time apt-get install -y python-dev
+time apt-get install -y libboost-python-dev
+time apt-get install -y python-numpy
+time apt-get install -y libgif-dev
+time apt-get install -y libpng-dev
+#time apt-get install -y libopenjpeg-dev
+time apt-get install -y libraw-dev
+time apt-get install -y libwebp-dev
+time apt-get install -y libfreetype6-dev
+#time apt-get install -y libjpeg-turbo8-dev
+time apt-get install -y dcmtk
+time apt-get install -y libavcodec-dev
+time apt-get install -y libavformat-dev
+time apt-get install -y libswscale-dev
+time apt-get install -y libavutil-dev
+time apt-get install -y locales
+time apt-get install -y opencolorio-tools
+time apt-get install -y ninja-build
```

Comment:
All these individual apt-get install calls add a significant amount of time to the build. It would be better to do something like:

```
apt-get install -y \
    package-a \
    package-b \
    package-c
```

Reply:
Oh? It never occurred to me that that doing them on separate lines would have any important performance impact. I assumed that the speed would mostly be limited by the download speed of the packages themselves and their subsequent install. Thanks for the tip!

Reply:
Yep. The actual call to "apt-get" takes a few seconds (on slow systems much longer) when you're just doing 1 or 2 package installs this way you don't notice it as it's just a couple seconds. But once you get into the 10s of packages the valuable seconds really add up.

Doing a `apt-get install -y package-a package-b package-c` instead causes apt-get to init only once and thus will be much faster if you are installing many packages.